core/base/abstractTriangulation/AbstractTriangulation.h

```diff
@@ -2858,6 +2858,30 @@ namespace ttk {
       return this->hasPreconditionedDistributedCells_;
     }
 
+    ttk::SimplexId inline getDistributedGlobalCellId(ttk::SimplexId localCellId,
```

Comment:
I'd suggest the following change here (to be consistent with the other functions of the triangulations):
`inline int getDistributedGlobalCellId(const SimplexId &localCellId, const int &cellDim, SimplexId globalCellId) const{`

Reply:
the general idea here is to have input arguments as const and output arguments as non const